client/components/explorer/sidebar/sidebar-tabs-directive.css

```diff
@@ -23,4 +23,10 @@
 .pk-sidebar-tab-label {
   padding-left: 32px;
   padding-top: 2px;
+  /*
+   * Ensure there's always enough room to fit the largest-used label,
+   * currently this is "Data Results" which is longer than "Dashboard".
+   * This avoids unsightly relayout when showing/hiding labels.
+   */
+  min-width: 107px;
```

Comment:
I've changed the approach to override .ng-hide styling to use visibility:hidden instead of display:none. PTAL.